apps/web/src/app/[locale]/aot-2023/_components/tiltable-card.tsx

```diff
@@ -197,10 +197,13 @@ export function TiltableCard({ index, challenge }: Props) {
           </p>
         </motion.div>
       </motion.div>
-      <motion.div
+      <motion.a
+        href={`/challenge/${challenge.slug}`}
         onClick={(e) => {
           e.stopPropagation();
+          e.preventDefault();
           router.push(`/challenge/${challenge.slug}`);
+          return false;
```

Comment:
If there is an `a` tag with an href now, can't we completely remove the `onClick` handler?

Reply:
wouldn’t that cause a full page reload since we’re not using the router

Reply:
yes. and i didn't use Link, because i was not sure what motion is doing here. But i can definitely flip to Link if prefered

Reply:
this code won’t live much longer so not a big deal